packages/metro/src/ModuleGraph/worker/collectDependencies.js

```diff
@@ -70,6 +77,8 @@ export type State<TSplitCondition> = {
 
 export type Options<TSplitCondition = void> = $ReadOnly<{
   asyncRequireModulePath: string,
+  /** Path to file for handling `require.context` statements */
+  requireContextModulePath: string,
```

Comment:
I don't think we're going to need a separate helper module a la `asyncRequire` - rather, any shared runtime logic for this would live in `require.js` directly.

Reply:
oops, this is from an earlier pass -- removing.